EOY 2024 (#376)
OpenSats/website: data/blog/2024-year-in-review.mdx

[Camelus](https://github.com/leo-lox/camelus),
[Captain's Log](https://github.com/nodetec/comet),
[dart_ndk + yana](https://github.com/relaystr/ndk),
[DVMDash](https://github.com/dtdannen/dvmdash),
[Formstr](https://github.com/abhay-raizada/nostr-forms),
[futr](https://github.com/futrnostr/futr),
[Groups](https://github.com/max21dev/groups),
[jesterui](https://github.com/jesterui/jesterui),
[Mostro](https://github.com/MostroP2P/mostro),
[ngit](https://github.com/DanConwayDev/ngit-cli),
[nostr-php](https://github.com/nostrver-se/nostr-php),
[nostr-relay-tray](https://github.com/CodyTseng/nostr-relay-tray),
[nostr-sdk-ios](https://github.com/nostr-sdk/nostr-sdk-ios),
[nostr-spring-boot-starter](https://github.com/theborakompanioni/nostr-spring-boot-starter),
[nostr.build](https://github.com/nostrbuild/nostr.build),
[nostrability](https://github.com/nostrability),
[Nostrmo](https://github.com/haorendashu/nostrmo),
[Nostrocket](https://github.com/nostrocket),
[nostroots](https://github.com/Trustroots/nostroots),
[nostter](https://github.com/SnowCait/nostter),
[Notedeck](https://github.com/damus-io/notedeck),
[Osty](https://github.com/ostyjs/create-osty/),

> [Safebox](https://github.com/trbouma/safebox)

Reviewer: ```suggestion
[Safebox](https://github.com/trbouma/safebox),
```
Missing comma

---
title: "OpenSats' 2024 Year in Review"
date: '2024-12-30'
tags: ['OpenSats', 'grants', 'donations', 'bitcoin', 'nostr', 'review']
draft: false
authors: ['dergigi', 'default']
images: ['/static/images/blog/66-opensats-2024-year-in-review.jpg']
summary: "365 days. Two funds. 200+ grants. Billions of sats."
---

Twenty-twenty-four. What a year! A lot has happened in the past 12 months, and
just like last year, we want to take the opportunity to pause, reflect, and
share our year in review. We want to look back at the announcements we made,

> share an update in regards to the donations we received, and give an overview of

Reviewer 2: ```suggestion
share an update regarding the donations we received, and give an overview of
```

how we allocated our funds. After that we will share updates to our grantee
statistics and provide an outlook of what the next year might bring.

- [Announcements](#announcements)
- [Donations](#donations)
- [Allocation](#allocation)
- [Grantees](#grantees)
- [Outlook](#outlook)

> But before we dive into the various graphs and numbers, let us spend a minute to

Reviewer 2: ```suggestion
Before we dive into the various graphs and numbers, let us spend a minute to
```

Author: I think I'll keep the but(t) 😜

Reviewer 3: Agree that best not to begin a sentence with "But"; for instance, I think this would be more correct: s/But/However, /

remind everyone—and most importantly ourselves—_why_ we are doing what we are

> doing.  _"I'm convinced that to give away a dollar effectively is harder than to

Reviewer 4: Steve Jobs once said, "I'm convinced...

More concise, plus immediately gives authority to the following words.

properly—is an incredibly hard problem.

Why set out to solve this problem in the first place? Why go through the trouble
of creating a non-profit entity? Why spend countless hours reviewing grant
applications, commits, reports, pull requests, and other software artifacts? Why
bother with open-source grants at all? Shouldn't this problem be solved by
companies, or direct crowd funding, or the state?

From the beginning, the vision of OpenSats was to create a sustainable
foundation that provides [consistent funding](/about) for free and open-source
projects, especially those who make bitcoin flourish. We don't want to be
dependent on quarterly profits nor on funding from the state, which is why we
chose a public charity as a vehicle.

As mentioned in our [mission statement](/mission), we believe that we can help
make bitcoin more accessible, robust, secure, and decentralized. Further, we
believe that the projects we support have the potential to revolutionize the way
we think about individual sovereignty and liberty—values that we hold in high
regard. We want to operate with as little overhead as possible, which is why we

> don't take a cut from donations to our funds. And we don't want any one person

Reviewer 5: remove "to our funds", or clarify what is meant by "our funds"

or small group to have control over funding or organizational decisions, which
is why we have a 9-person board that is the final arbiter of all decisions.

We don't claim to have all the answers, and we don't claim that what we are
doing is frictionless or perfect. We are bound to make mistakes, and we will

> always be prone to false positives (funding projects that shouldn't receive

Reviewer 5: probably can leave out the comment about "false positives (funding projects that shouldn't receive....", that sounds bad

Author: I'll rephrase that.

we thought they would be) as well as false negatives (not funding projects that
should receive funding). We strive to gradually and continually improve our
processes, and we hope to get better at what we are doing with every passing
year. That said, we are incredibly pleased to see that our structure and
processes are working, and we believe this year in review is a testament to
that.

---

> ## Announcements

Reviewer 4: To avoid situations that announce announcements, I recommend renaming this section something like "Scope of our Grants" or "Scope and Focus of Our 2024 Grants".

Reviewer 4: If you want to stick to one word titles, consider calling this section "Grantees" and renaming the other section with the same name as "Growth".

> At the beginning of the year, we announced long-term support for [René

Reviewer 5: maybe define LTS somewhere?

Pickhardt](/blog/rene-pickhardt-receives-lts-grant) and
[0xB10C](/blog/0xB10C-receives-lts-grant), two researchers/developers with a

> major track record, both doing important work related to bitcoin infrastructure.

Reviewer 5: "major track record of doing important work..."

Reviewer 6: "proven track record" also solid

major track record of doing important work related to bitcoin infrastructure.

> Soon after we announced long-term support for [Bruno

Reviewer 4: "Soon after we also provided long-term support Bruno Garcia...". I recommend these changes because they are more action oriented and to avoid sounding like OpenSats is announcing announcements. If you are interested in these changes I can make more throughout this section.

as a [load-bearing internet
person](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors)
by our estimation. In May [Shashwat
Vangani](/blog/shashwat-vangani-receives-lts-grant) joined our roster of LTS
grantees, followed by [Calle](/blog/cashu-calle-receives-lts-grant) in June, and
[Jason Donenfeld](/blog/jason-donenfeld-lts-grant)
&
[Jon Atack](/blog/jon-atack-receives-lts-grant)
in September. The last two long-term
support grants in 2024 were awarded to [Dusty
Daemon](/blog/dusty-daemon-receives-lts-grant) and
[m1sterc001guy](/blog/m1sterc001guy-receives-lts-grant), as announced in October
and November, respectively. This brings the total number of grantees receiving
long-term support from our General Fund to 18, adding to the 9 announced in 2023

> ([Marco](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors),

Reviewer: Just flagging this line in case the link does actually exist? Could not find it in the codebase though.

Author: That's the correct link, we announced his LTS with the [announcement of the program](https://opensats.org/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors):

> The first recipient of an OpenSats LTS Grant is Marco Falke, a long-term maintainer and contributor of Bitcoin Core [...]

([Marco](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors),
[Josi](/blog/josi-baker-receives-opensats-lts-grant),
[Sjors](/blog/sjors-provoost-receives-opensats-lts-grant),
[Vasil](/blog/vasil-dimov-receives-lts-grant),
[Gleb](/blog/gleb-naumenko-receives-lts-grant),
[Matt](/blog/matt-morehouse-lightning-security-lts-grant),
[Will](/blog/will-clark-receives-lts-grant-for-bitcoin-core),
[Furszy](/blog/furszy-receives-lts-grant-for-bitcoin-core), and
[Tobin](/blog/tobin-harding-receives-lts-grant-for-rust-bitcoin)).

![](/static/images/blog/2024-eoy/lts-grantees.png)

> Speaking of long-term support, in March we announced the expansion of our LTS

Reviewer 4: "In March we expanded our long-term support program to include nostr developers."

More concise.

Casarin](/blog/jb55-receives-lts-grant) aka jb55 in June, [Yuki
Kishimoto](/blog/yuki-receives-lts-grant),
[Hodlbod](/blog/hodlbod-receives-lts-grant), and
[fiatjaf](/blog/fiatjaf-receives-lts-grant) in July,
[Kieran](/blog/kieran-receives-lts-grant),
[Vitor](/blog/vitor-pamplona-receives-lts-grant),
[Daniele](/blog/daniele-receives-lts-grant), and [Alex
Gleason](/blog/alex-gleason-receives-lts-grant) in August, [Mike
Dilger](/blog/mike-dilger-receives-lts-grant) in September, and
[Greenart7c3](/blog/greenart7c3-receives-lts-grant) in October. In addition to
these nostr LTS grants, we announced 5 waves of nostr grants—in
[July](/blog/nostr-grants-july-2024), [August](/blog/nostr-grants-august-2024),
[September](/blog/7th-wave-of-nostr-grants),
[November](/blog/8th-wave-nostr-grants), and
[December](/blog/9th-wave-of-nostr-grants)—giving a total of 35 grants to some
of the most impactful projects in the nostr ecosystem. We also shared some
[highlights of
#nostrdesign](/blog/nostr-design-highlights-may) and
recently committed to [doubling down on nostr](/blog/doubling-down-on-nostr).

In addition to these LTS and nostr grants, we announced [5 waves of bitcoin

> grants](/tags/wave), giving short- and medium-term grants to 37 high-impact

Author: [Suspended hyphen](https://english.stackexchange.com/a/113433) is correct in that case, I believe.

> grants](/tags/wave), giving short- and medium-term grants to 37 high-impact

Reviewer: Oh wow. Learning new stuff here :laughing:

the latter received a grant from us this year. [Strengthening network privacy
with Tor](/blog/tor-receives-support-grant) is crucial, as is the [development
and maintenance of WireGuard](/blog/jason-donenfeld-lts-grant). We deeply
believe that everyone should be able to explore and use the internet privately
and securely and that privacy and free speech must be defended online through
free software and open networks.

Bitcoin is one of the most powerful open networks to date, and we believe that
this will remain to be the case for decades to come. By [renewing our commitment
to bitcoin](/blog/renewing-our-commitment-to-bitcoin) we want to make explicit
that which should go without saying: we are here to support free and open-source
projects that help bitcoin flourish, especially those that don't have a clear
path towards direct monetization. Protocol development is an area that will be
reliant on grants for the foreseeable future, which is why we will continue to
[support core developers](/tags/core)—whether it’s seasoned veterans or
[up-and-coming devs](/blog/caring-for-bitcoin-core), as announced in August. If

> you are a developer who wants to quit their soul-sucking fiat job to work on

Reviewer 5: "their" should be "your" (match pronouns)

because of donors like you. We want to express our deep gratitude to everyone
who donated to [our funds](/funds) as well as our [operations
budget](/funds/ops). If you like what we do and want to support our work, please
consider [making a donation](/donate). Thank you.

---

## Donations

In 2024 we received 929 individual donations for a total sum of \$23.6M. The
largest three were a generous \$1,000,000 gift from the [Reynolds
Foundation](/blog/opensats-receives-one-million-from-reynolds-foundation), an
incredibly generous \$21M donation from
[#startsmall](/blog/opensats-receives-additional-funding-of-dollar21m-from-startsmall),
and an anonymous donation of \$1,000,000 made in bitcoin directly.

As announced at the beginning of the year, OpenSats is to receive [recurring
donations from Bitwise Bitcoin ETF
profits](/blog/recurring-donations-from-bitwise-bitcoin-etf-bitb), along with
Brink and the Human Rights Foundation’s Bitcoin Development Fund. This was
followed by a [recurring donation commitment from Build Asset
Management](/blog/donation-commitment-from-build-asset-management), which we
announced at the end of the second quarter, as well as a recurring commitment

> announced by Cathy Wood, who [pledged to donate a fixed percentage of Ark’s

Reviewer 5: *Cathie Wood

Author: Damn! Good catch, thank you 🧡

amounts, around ~95% of money donated by the general public is directed towards
our General Fund.

<center>
    <DonateToGeneralFundButton/>
</center>

---

## Allocation

The purpose of our two funds is to support free and open-source projects that
are aligned with [our mission](/mission). The [Nostr Fund](/funds/nostr) is
dedicated towards nostr projects, a protocol that is just coming out of its
infancy. The [General Fund](/funds/general) is dedicated towards all other free
and open-source projects and contributors, especially bitcoin-related projects
and projects that help bitcoin flourish.

As of December 2024, we have allocated over \$12M in grants from the General
Fund, and close to \$9M in grants from The Nostr Fund. In terms of money
allocated [year-over-year](/blog/2023-year-in-review#allocation) this is an
increase of 67% on the nostr side, and 81% on the bitcoin side. When looking at
total allocation in USD terms, our two funds are at ~89% and ~52% allocated,
respectively.

![](/static/images/blog/2024-eoy/allocation.png)

As mentioned in [last year’s report](/blog/2023-year-in-review), we suspect that

> the distinction between “bitcoin” and “nostr” will get more blurry over time, as

Reviewer 5: "blur" instead of "get more blurry"

nostr matures and gets integrated into a variety of bitcoin software across the
stack. This is already happening, especially on the eCash front. Are [Cashu
addresses](/blog/nostr-grants-july-2024#npubcash) a Lightning thing, a bitcoin
thing, or a nostr thing? Are wallets that use nostr for social payments nostr
clients, or bitcoin wallets? And what about P2P exchanges that use nostr to
publish their order books? It is obvious to us that various concepts and
technologies are currently merging, and we expect this cross-pollination of
ideas to continue.

> With all that said, nostr is still an incredibly young network. It is just

Reviewer 5: "It is just coming out of its infancy" is repetitive. Just say "incredibly young network, which is why..."

Out of the ~\$23M donated towards our General Fund during the lifetime of the
fund, we have allocated ~\$12M towards free and open-source projects so far. In
terms of gross amounts the top five categories that we gave grants to are
Bitcoin Core, Lightning, eCash, Education, and Privacy.

![](/static/images/blog/2024-eoy/bitcoin-distribution.png)

Our support for Bitcoin Core is mostly constituted of long-term support grants
for core devs, namely:
[Marco](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors),
[Josi](/blog/josi-baker-receives-opensats-lts-grant),
[Sjors](/blog/sjors-provoost-receives-opensats-lts-grant),
[Vasil](/blog/vasil-dimov-receives-lts-grant),
[Gleb](/blog/gleb-naumenko-receives-lts-grant),
[Furszy](/blog/furszy-receives-lts-grant-for-bitcoin-core),
[Will](/blog/will-clark-receives-lts-grant-for-bitcoin-core),
[0xB10C](/blog/0xB10C-receives-lts-grant),
[Bruno](/blog/bruno-garcia-receives-lts-grant), and
[Jon](/blog/jon-atack-receives-lts-grant). The last three were added to our LTS
roster in 2024. In addition to these long-term grants we gave smaller grants to
[three up-and-coming core developers](/blog/caring-for-bitcoin-core).

> 

Reviewer 5: Little bit repetitive to discuss LTS twice in one blog post. Could probably merge the earlier section on LTS with this section here.

[Clams](https://github.com/clams-tech),
[Krux](https://github.com/selfcustody/krux),
[Vexl](https://github.com/vexl-it/vexl),
[satsigner](https://github.com/satsigner/satsigner),
[Bitaxe](https://github.com/skot/bitaxe),
[Braidpool](https://github.com/pool2win/braidpool/),
[Utreexo](https://github.com/utreexo/utreexo),
[Fedimint](https://github.com/fedimint/fedimint),
[macadamia](https://github.com/zeugmaster/macadamia),
[Krux-Installer](https://github.com/selfcustody/krux-installer),
[Krux](https://github.com/selfcustody/krux),
[Cove](https://github.com/bitcoinppl/cove),
[secp256k1.swift](https://github.com/21-DOT-DEV/swift-secp256k1),
[BDK](https://github.com/bitcoindevkit),
[Labelbase](https://github.com/Labelbase/Labelbase/),
[Krux](https://github.com/selfcustody/krux),
[dlcdevkit](https://github.com/bennyhodl/dlcdevkit),
[Cove](https://github.com/bitcoinppl/cove),
[StratumV2](https://github.com/stratum-mining/benchmarking-tool),
[Floresta](https://github.com/vinteumorg/floresta),
[Vexl](https://github.com/vexl-it/vexl), [LNbits](https://github.com/lnbits),
[El Tor](https://bitbucket.org/eltordev/eltor), [BTCPay
Server](https://github.com/btcpayserver), and

> [Raspiblitz](https://github.com/raspiblitz/raspiblitz).

Author: Oh wow, it must have been very late when I made that list. Thank you. Alphabetical order it is 🙏

One of our goals for 2025 is to receive multiple recurring commitments to ensure
that our long-term support programs are well-funded. After all, [our
mission](/mission) is to maintain a sustainable ecosystem of funding for free
and open-source projects and contributors. We hope that our track record will
incentivize those who rely on bitcoin to give to OpenSats periodically, both to
one of our funds as well as to our operations budget, which we use to
independently fund our small team. We are one of the few charities that don’t
take a cut from donations, so if you think we are doing good work and you want
us to continue to exist, please continue to support us:

<center>
    <DonateToOperationsButton/>
</center>

Our hope is to scale our grant-giving efforts considerably going forward, and we
will share some thoughts on this matter at the beginning of the new year. In the
best case, we want to see many more organizations like OpenSats, as we believe
that specialization and a broader support network are paramount as the various
ecosystems grow and specialize. We don’t have a crystal ball of course, but from

> our perspective, it isn’t hard to imagine that nostr and ecash are here to stay

Reviewer: ```suggestion
our perspective, it isn’t hard to imagine that nostr and eCash are here to stay
```

1 of 2 mentions of  `ecash` not using a capitalized `C`

and are bound to grow, and it isn’t outlandish to think that open-source
hardware will gain in importance as well. Bitcoin will remain to be of the
utmost importance of course, and as it enters the political arena of the world
stage we expect to see multiple non-profits being created to support open-source
developers across all kinds of jurisdictions. If you are thinking about creating
such an organization don’t hesitate to [reach out to us](/contact).

Last but not least, we want to dedicate this final paragraph to all of you.
Whether you are a donor, grantee, future applicant, or simply an interested
reader who is interested in bitcoin and freedom tech—the revolution that is

> currently happening is only happening because of you. We are incredibly thankful

Reviewer 3: (several uses of the word "incredibly" in this post, perhaps vary with other words)

Author: Well spotted! Down to one now. 

Thank you for the in-depth review and the great suggestions, I truly appreciate it 🙏🧡

for your time, care, and attention. You could be spending it on anything else,
but you choose to spend it on bitcoin and freedom tech. Thank you. We are beyond
grateful for the support we received, both monetarily as well as in the form of

> community and [volunteer efforts](/about#volunteers), and it's exhillerating to

Reviewer 4: replace _exhillerating_ with _exhilarating_
